packages/node_modules/@webex/internal-plugin-encryption/src/kms-batcher.js

```diff
@@ -25,7 +25,7 @@ const KmsBatcher = Batcher.extend({
     return Promise.all(event.encryption.kmsMessages.map((kmsMessage) => new Promise((resolve) => {
       /* istanbul ignore else */
       if (process.env.NODE_ENV !== 'production') {
-        this.logger.info('kms-batcher:', kmsMessage.body);
+        this.logger.info('kms-batcher:');
```

Comment:
Do we need to be concerned with PII if it is not production code?

Reply:
@mwegman - security team requested another review for all services/clients to make sure there is no PII in logs.  Discussed with @andrewholsted and other members and concluded to remove PII for all env. 
we can discuss once again within the team, and add conditional logging.

Reply:
I'm fine with that. Just wanted to make sure.

Reply:
SDK core engineering team needs to discuss more about this change within. Will update you shortly.